packages/expo-updates/e2e/fixtures/project_files/.fingerprintignore

```diff
@@ -0,0 +1 @@
+**/ios/Gymfile
```

Comment:
@Kudo - this Gymfile seems to be auto-generated by EAS Build when not present. This should only affect our e2e setup (it shouldn't affect normal usage) since we also calculate the fingerprint at time of manifest serving (which is abnormal).